client/src/assets/scss/base.scss

```diff
@@ -1,40 +1,98 @@
 /*
- * Base Styles
+ * Base styles
  */
 
 @import "theme.scss";
 
-/* @font-face should go here */
+/*
+ * Fonts
+ */
+@font-face {
+  font-family: 'BCSans';
+  font-style: normal;
+  font-weight: normal;
+  src: url('~@/assets/fonts/BCSans-Regular.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'BCSans';
+  font-style: italic;
+  font-weight: normal;
+  src: url('~@/assets/fonts/BCSans-Italic.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'BCSans';
+  font-style: normal;
+  font-weight: bold;
+  src: url('~@/assets/fonts/BCSans-Bold.woff') format('woff');
+}
 
-/* HTML element styles should go here */
+@font-face {
+  font-family: 'BCSans';
+  font-style: italic;
+  font-weight: bold;
+  src: url('~@/assets/fonts/BCSans-BoldItalic.woff') format('woff');
+}
+
+/*
+ * HTML element styles
+ */
 .h6, h6 {
   color: $dk-text;
   font-size: 1rem;
   line-height: 20px;
-  font-weight: 700;
-}
-/* global styles should go here */
-.pr-75 {
-  padding-right: 75px !important;
-}
-.pl-75 {
-  padding-left: 75px !important;
-}
-.action {
-  color: $error !important;
+  font-weight: bold;
 }
+
+/*
+ * Global styles
+ */
 .approved {
   color: $approved !important;
 }
+
 .error-message {
-  color: $error;
+  color: $app-red !important;
+}
+
+// the 3 name colours
+.name-approved {
+  color: $app-green !important;
+}
+.name-rejected {
+  color: $app-red !important;
+}
+.name-ne {
+  color: $text !important;
 }
 
-.content-container {
-  min-width: 960px !important;
-  max-width: 1300px !important;
-  margin-left: auto;
-  margin-right: auto;
-  padding-left: 0;
-  padding-right: 0;
+// background colours
+.bg-light-gray {
```

Comment:
We have too many different gray background (and gray borders).

If you get a chance, please reduce this number.